styles/jupyter.css

```
@@ -1,47 +1,43 @@
 .font-system {
   font-family: Menlo, Consolas, 'DejaVu Sans Mono', monospace;
 }
+
 .jupyter-error {
-  background-color: rgb(255, 221, 221);
+  @apply bg-red-100;
 }
+
 /* Hide the output prompts from jupyter by default */
 .jp-OutputPrompt {
-  display: none;
+  @apply hidden;
 }
+
 /* Pandas tables */
 /* Here we use important to overwrite mt from typography.css */
+
 table.dataframe {
-  border: none;
-  border-collapse: collapse;
-  border-spacing: 0;
-  color: black;
-  font-size: 1em;
-  table-layout: fixed;
-  margin: 0 !important;
+  @apply border-none border-collapse border-spacing-0 text-xs table-fixed m-0 !important;
```

Check whether I've used the !important tag correctly, because I think it might be overriding something else (thus why black text was in both themes before a recent commit)

---

I messed up a bit with last 4 commits, but the diff is as intended

---

I think this should be `!m-0` ([docs](https://v3.tailwindcss.com/docs/configuration#important-modifier)).

```suggestion
  @apply border-none border-collapse border-spacing-0 text-xs table-fixed !m-0;
```